evennia/accounts/bots.py

```diff
@@ -68,21 +67,6 @@ def at_repeat(self):
         for session in _SESSIONS.sessions_from_account(self.account):
             session.update_session_counters(idle=True)
 
-    def at_server_reload(self):
-        """
-        If server reloads we don't need to reconnect the protocol
-        again, this is handled by the portal reconnect mechanism.
-
-        """
-        self.db.started = True
-
-    def at_server_shutdown(self):
-        """
-        Make sure we are shutdown.
-
-        """
-        self.db.started = False
-
```

Comment:
The purpose of these checks were to make sure you didn't re-initialize the bot even if the script was refreshed multiple times. With this change, it seems like this could start over and over and redo the intialization as long as there are no sessions connected to the server?

Reply:
That's true, yeah. I was having some issues with `.db.started` not being set back to False when I had to stop the server, and looking at the various bots it seemed as though the main purpose of initializing the bot (that is, running `start()`) is to connect the sessions, so that seemed to me to be an equivalent check. e.g. if it's not connected, start it; if it's connected, it's already been started. I could very well have misinterpreted how that was working, though.

Reply:
Hm. Let's try this simplification and see how it goes; it _should_ be okay, I'm just always a wary of changing functionality like this in a parent class affecting many subclasses not under test.

Reply:
Yeah, I debated it for a while, but determined that if it wasn't working when I was using it the same way as the other bots, it probably wouldn't work for them either.

I'd like to add unit tests for all of the chat relay bot accounts across the board over the next month or two, but that'll take me a while so I wanted to get this one working and in first. If there are any issues with this revision, I *should* catch it then.